anomalib/models/dfm/dfm_model.py

```diff
@@ -39,13 +39,13 @@ def __init__(self):
     def fit(self, dataset: Tensor) -> None:
         """Fit a Gaussian model to dataset X.
 
-            Covariance matrix is not calculated directly using:
-                C = X.X^T
-            Instead, it is represented in terms of the Singular Value Decomposition of X:
-                X = U.S.V^T
-            Hence,
-                C = U.S^2.U^T
-            This simplifies the calculation of the log-likelihood without requiring full matrix inversion.
+        Covariance matrix is not calculated directly using:
+            C = X.X^T
```

Comment:
maybe we could have these equations within quotes to properly render in the docs

Reply:
like `C = X.X^T`?